tests/components/zwave_js/test_diagnostics.py

```diff
@@ -42,7 +42,11 @@ async def test_config_entry_diagnostics(
         for node in nodes:
             assert "location" not in node or node["location"] == REDACTED
             for value in node["values"]:
-                if value["commandClass"] == 99 and value["property"] == "userCode":
+                if (
```

Comment:
I'd store the resulting json diagnostic data in a fixture file instead and just assert that the whole contents is the same as expected.